Random Battle Updates

ascriptmaster opened the pull request:
- Reject Quick Attack with Feint
- Remove Return, Quick Attack, Feint from regular Pinsir
- Remove Feint from Hitmonchan, Hitmonlee
- Add Fire Punch to Hitmonchan
- Replace Stone Edge with Rock Slide for Doubles

TheImmortal commented:
What's Feint doing on regular Pinsir? Remove it. And probably remove Stone Edge from Doubles on both.

ascriptmaster commented:
Removed. I removed Quick Attack as well.

I also took Feint off Hitmonchan/Hitmonlee, and added Fire Punch to Hitmonchan.

Hitmontop can keep Feint, because it has Technician.
